### PR TITLE
Add Initial Version

### PR DESCRIPTION
**Pull request by sfjc2000-LA**
### Overview
* Making the weather application
* To tell the current weather
* In specific weather, it will be alarmed

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import axios from "axios";
+import Weather from "./Weather";
+
+const API_KEY = "d19f3d0e91935a8673ca99d5dc95c98d";
+
+export default class extends React.Component {
+  state = {
+    isLoading: true
+  };
+  getWeather = async (latitude, longitude) => {
```
Comment (jelee0118) [inline App.js:14]: When a function starts, put a new line in it 🙏

File: Weather.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    subtitle: "You should take your mask"
+  },
+  Haze: {
+    iconName: "weather-hail",
+    gradient: ["#4DA0B0", "#D39D38"],
+    title: "Haze",
+    subtitle: "Just don't go outside."
+  }
+};
+export default function Weather({ temp, condition }) {
```
Comment (jelee0118) [inline Weather.js:66]: Put a new line, please 😄

**Review by jelee0118 — COMMENTED** [review]
@sfjc2000-LA 
GREAT JOB jae hyun 👍 👍 👍 👍 👍 
I made a comment about new line 😅, so please correct it.
Because coding inventions are important to coder 😎 🚀

File: App.js (hunk 1 above)
Comment (sfjc2000-LA) [reply App.js:15]: I changed it.

Comment (sfjc2000-LA) [reply App.js:15]: I changed it.

File: Weather.js (hunk 2 above)
Comment (sfjc2000-LA) [reply Weather.js:67]: This too.

**Review by jelee0118 — APPROVED** [review]
Nice work 👍 👍 👍 👍 
Now merge it please 🙏

**Pull request merged**